lib/src/internal/test_helpers.dart

```diff
@@ -25,4 +26,8 @@ class BuiltCollectionTestHelpers {
   static BuiltSet<int> overridenHashcodeBuiltSet(
           Iterable iterable, int hashCode) =>
       new OverriddenHashcodeBuiltSet<int>(iterable, hashCode);
+
+  static BuiltSetMultimap<int, String> overridenHashcodeBuiltSetMultimap(
+          map, int hashCode) =>
```

Comment:
> declare it as dynamic so it doesn't look like you forgot to declare the type?

https://www.dartlang.org/articles/style-guide/#avoid-annotating-with-dynamic-when-not-required